content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,6 +17,10 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+## Specify the custom folder
+
+The default custom folder is `GITEA_HOME/custom`, and you can set it by the env `GITEA_CUSTOM`.
+
```

Comment:
I will. I'm checking all `os.Getenv` in Gitea code and extract them. Then I list them on a page to explain what they do in the code. I will put the changes in #111, so that we can talk about it.